README.md

```diff
@@ -1,3 +1,31 @@
+# D2BS with HotFixes and extended functionality 
```

Comment:
I think you dont want to add this text in the original, as if its in the original it isnt a fork anymore